enumeratum-core/src/main/scala/enumeratum/EnumEntry.scala

```diff
@@ -167,6 +175,9 @@ object EnumEntry {
     */
   trait LowerCamelcase extends EnumEntry with Camelcase with Uncapitalised
 
+  abstract class MultiEnum(override val entryName: String,
```

Comment:
I don't think this is needed (at least not here); looks like something you want in your tests.

Reply:
Fixed.